#From: Thamognya Kodi <contact@thamognya.com>
#Date: Fri, 22 Apr 2022 12:07:08 +0000
#Subject: [PATCH] patch to add a new install variable to 
#Makefile to install it

#This Patch adds an install variable to the Makefile to install the
#package's script as the distfiles in https://dev.gentoo.org/~radhermit/distfiles/
#are not maintained.

--- ./Makefile
+++ ./testing/Makefile
@@ -1,4 +1,5 @@
 # help me remember 'python setup' commands
+INSTALL_DIR="/usr/bin/"
 
 help:
 	pypi - build sdist/bdist_egg and upload to pypi
@@ -7,6 +8,9 @@
 pypi:
 	python3 setup.py sdist bdist_egg upload
 
+install:
+	cp -r ./terminal-colors "${INSTALL_DIR}/terminal-colors.py"
+
 release:
 	@echo git tag -a vX.XX.X -s -f
 	@echo git push --tags
